.eslintrc.js

```diff
@@ -100,7 +130,10 @@ module.exports = {
         'babel/new-cap': ['error', { capIsNewExceptionPattern: '^styled.' }],
         '@typescript-eslint/no-unsafe-assignment': 'off',
         '@typescript-eslint/no-unused-vars': ['error'],
-        // '@typescript-eslint/no-unsafe-return': 'off',
+        '@typescript-eslint/no-unsafe-member-access': 'off',
```

Comment:
what was the need to turn them off? they were not causing any errors previously. what changed?